include/swift/SIL/GraphOperationInfo.h

```diff
@@ -55,6 +55,16 @@ struct GraphOperationInfo {
     /// to an attribute.
     ///
     /// Written as named argument with "$tensor" suffix.
+    ///
+    /// Because of how lowering is implemented,
```

Comment:
What aspect of lowering are you referring to?

Reply:
I'll clarify the comment